File: test/Coverage/ast-printing.m
// RUN: %lfort_cc1 -fsyntax-only -fobjc-exceptions %s
// RUN: %lfort_cc1 -ast-print -fobjc-exceptions %s
// RUN: %lfort_cc1 -ast-dump -fobjc-exceptions %s

#include "objc-language-features.inc"

File: test/Coverage/ast-printing.c
// RUN: %lfort_cc1 -fsyntax-only %s
// RUN: %lfort_cc1 -ast-print %s
// RUN: %lfort_cc1 -ast-dump %s
// RUN: %lfort_cc1 -print-decl-contexts %s

#include "c-language-features.inc"
